lib/mirage.ml

```diff
@@ -471,10 +472,14 @@ module Entropy = struct
     append_main "module %s = %s" (module_name t) (construction t) ;
     newline_main () ;
     append_main "let %s () =" (name t);
-    append_main "  %s.connect %s" (module_name t) (id t);
+    append_main "  %s.connect %s >>= function" (module_name t) (id t);
+    append_main "  | `Error e -> %s" (driver_initialisation_error "entropy");
+    append_main "  | `Ok entropy ->";
+    append_main "  %s.handler entropy Nocrypto.Rng.Accumulator.add_rr >>= fun () ->" (module_name t);
```

Comment:
I'm eager to have a release asap - but really don't want to mess this up. better safe (and release slightly later) than sorry (and introduce various kinds of security issues).
Supporting TLS out of the box in MirageOS would be a great time to have a security contact mail address and release security advisories if there are security relevant issues found (and describe our practices somewhere on http://openmirage.org).